runtime/interpreter/value.go

```diff
@@ -6049,19 +6049,25 @@ func (v UFix64Value) ConformsToDynamicType(_ *Interpreter, dynamicType DynamicTy
 // CompositeValue
 
 type CompositeValue struct {
-	Location            common.Location
-	QualifiedIdentifier string
-	Kind                common.CompositeKind
-	Fields              *StringValueOrderedMap
-	InjectedFields      *StringValueOrderedMap
-	ComputedFields      *StringComputedFieldOrderedMap
-	NestedVariables     *StringVariableOrderedMap
-	Functions           map[string]FunctionValue
-	Destructor          FunctionValue
-	Owner               *common.Address
-	destroyed           bool
-	modified            bool
-	stringer            func() string
+	// Storable properties
+	location            common.Location
+	qualifiedIdentifier string
+	kind                common.CompositeKind
+	fields              *StringValueOrderedMap
+
+	InjectedFields  *StringValueOrderedMap
+	ComputedFields  *StringComputedFieldOrderedMap
+	NestedVariables *StringVariableOrderedMap
+	Functions       map[string]FunctionValue
+	Destructor      FunctionValue
+	Owner           *common.Address
+	destroyed       bool
+	modified        bool
+	stringer        func() string
+
+	// Raw-content cache for decoded values. Maintains the meta info and field content separately.
+	metaContent   []byte
+	fieldsContent []byte
```

Comment:
Would be good to have comments here describing what is considered part of each of these two parts